auditlog/README.md

```diff
@@ -73,6 +73,49 @@ $ docker run -it --rm \
     -C -o beginning -q -u -t dbserver1.inventory.vegetable.enriched | jq ."
 ```
 
+## Administrate missing events
+
+There could be situations when there are missing transaction context data, for example
```

Comment:
Probably worth pointing out here that this service is based on Kogito?

Reply:
added few lines about kogito here

Reply:
Nice!